src/components/schemas/expense.yaml

```diff
@@ -769,4 +769,147 @@ expense_creation_from_receipt_in:
         This field takes the base64 encoded representation of the file.
       example: JVBERi0xLjQKMSAwIG9iago8PAovVGl0bGUgKP7/KQovQ3JlYX...
     source:
-      $ref: './fields.yaml#/source'
\ No newline at end of file
+      $ref: './fields.yaml#/source'
+
+expense_check_policies_in:
+  type: object
+  additionalProperties: False
+  required:
+    - claim_amount
+    - spent_at
+  properties:
+    id:
+      $ref: './fields.yaml#/id_string'
+      description: |
+        Expense id provided by Fyle during the creation of expense.
+    spent_at:
+      $ref: './fields.yaml#/spent_at'
+    source:
+      $ref: './fields.yaml#/source'
+    merchant:
+      $ref: './fields.yaml#/merchant'
+    foreign_currency:
+      $ref: './fields.yaml#/foreign_currency'
+    foreign_amount:
+      $ref: './fields.yaml#/foreign_amount'
```

Comment:
Override this. It is saying some - object was initially created.
But it should be expense.